charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,63 @@
+
+# Metadata Schema
+
+## Description
+
+The HCA Metadata schema committee creates and maintains the Human Cell Atlas metadata schema in JSON format. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure the FAIRness* of the HCA data [The FAIR Guiding Principles for scientific data management and stewardship](https://www.nature.com/articles/sdata201618). 
+
+## Objective
+
+1. Ensure that HCA data are findable and the schemas are computationally queryable.
+2. Ensure that HCA metadata supports reproducibility and reusability.
+3. Ensure that HCA metadata has a consistent representation and is computationally queryable.
+4. Define JSON schema which enable objectives 1 and 2.
+5. Specify the entity structure for HCA metadata schema.
+6. Define style guides and release process.
+7. Ensure the defined metadata schema provides a positive user experience for those who read and write them.
+
+## In-scope
+
+* Defining the specification for the schema and the the schema itself.
+* Incorporate community feedback to extend and improve the schema.
+* Ensuring the schema is compatible with open source software packages supporting the same draft of the JSON schema.
+* Provide a test framework to enable community members to test if their proposed extensions are functional and meet our style guides.
+* Providing exemplar data in the schema format.
```

Comment:
I think you mean exemplar data in JSON format that validates against the schema?